include/sc2api/sc2_unit.h

```diff
@@ -214,6 +215,17 @@ class UnitPool {
     void ClearExisting();
     bool UnitExists(Tag tag);
 
+    inline const std::vector<Unit const*>& GetNewUnits() const noexcept { return units_newly_created_; };
```

Comment:
Style:
`inline` is not ended here as the methods defined in the *.h field are lined by default, if I not mistaken.

Reply:
```suggestion
    const std::vector<Unit const*>& GetNewUnits() const noexcept { return units_newly_created_; };
```

Reply:
I agree with all these comments and style comments as well.. Should I do them here with suggestions? or by resubmitting a PR?

Reply:
You should do combination of `git rebase -I` (to add changes) + `git push --force` (to replace the old commits with the changed version). There is no need in recreating the PR.

Reply:
I'm not familar enough PRs and forks know what this actually does. My repository is a fork of yours, and I created the PR from my remote repository on github. If I do the above git commands from from my local repository I assume it will only alter my own fork on github, right?

Reply:
>If I do the above git commands from from my local repository I assume it will only alter my own fork on github, right?

Yes, that's why it is safe.

Reply:
scary stuff ;) I pushed changes to the branch on github that the PR was created from. Does this mean you can see them automatically now? 
That should be all the changes from all your comments for this PR. If I missed anything let me know.